sink/sink-centosci.yaml

```diff
@@ -39,6 +39,13 @@ items:
             - name: config
               mountPath: /run/config
               readOnly: true
+            resources:
+              limits:
+                memory: 1G
+                cpu: 2
+              requests:
+                memory: 200Mi
+                cpu: "100m"
```

Comment:
I must say, that I love how 3 values with units are all defined differently. One is in 10 base, one in 2 base and another one is string in `"` :)

Reply:
You are right, I cleaned that up.